### ISSUE TITLE
Marketplace not displaying projects in "Add to Project" modal

### ISSUE THREAD

**Issue by ebrammer** [FlutterFlow/flutterflow-issues#1487]
### Has your issue been reported?

- [X] I have searched the existing issues and confirm it has not been reported.
- [X] I give permission for members of the FlutterFlow team to access and test my project for the sole purpose of investigating this issue.

### Current Behavior

In the new implementation of Marketplace, whenever you click on "+ Add to Project" the dialog comes up with "Add to Project" and an empty modal.  Last night there was a single project there (could not tell why that one showed versus any of the others). I was able to successfully add a custom function to that project, but now not even that is showing. No projects are showing.

### Expected Behavior

I expect to see a list of all of my projects so that I can add a marketplace item to one of them.

### Steps to Reproduce

1. Go to Marketplace
2. Click on "Dashboard"
3. Click on "+ Add to Project"

### Reproducible from Blank

- [X] The steps to reproduce above start from a blank project.

### Bug Report Code (Required)

n/A

### Context

Unable to use templates added to account.

### Visual documentation

![image](https://github.com/FlutterFlow/flutterflow-issues/assets/5923757/d658ef0a-5e5b-4fe5-a580-c25dbf93abe0)

### Environment

```markdown
- FlutterFlow version: V.4 (released Sept 19., 2024)
- Platform: Web
- Browser name and version: Chrome Version 116.0.5845.140 
- Operating system and version affected:  MacOS High Sierra 10.13.6
```

**Comment by hariprasadms** [comment]
Hi @ebrammer - Thanks for submitting the issue. I have tested the issue. It is working fine. Not able to reproduce the steps.
Is this resolved for your now ? Please update the status. Thanks

**Comment by ebrammer** [comment]
Hari,

No. It is still doing it. Here is a video of the issues:

https://www.loom.com/share/4a31dd210d254a399059b01de2004600?sid=6a3a32b9-4e8c-48b5-8f86-8682bda69ff2

Evan

**Comment by hariprasadms** [comment]
Hi, Thanks for the video details. Can you confirm you got projects in your account? Update issue with project bug report code please.

**Comment by ebrammer** [comment]
Yes. I have about ten projects. I can’t do a bug report code since it’s
marketplace takes you out of the app. What would you suggest?

**Comment by stasnepo** [comment]
I had similar experience as @ebrammer
Found the root cause and the "temporary workaround"
@hariprasadms Initial tutorial App, and a custom Template App from Marketplace, are not being displayed under "Add to Project".
Creating a brand new blank project will appear under "Add to Project".

**Comment by ebrammer** [comment]
It works! Thank you so much.

**Comment by harshmehta15** [comment]
@ebrammer @hariprasadms @katherineqian 

Still facing the same issue. Fix has not worked for me.
https://www.loom.com/share/14d007c105ea4fb4ba7cf55e6d0392a2?sid=ee6b41d4-293c-4081-b7e7-3ec51c958c74

UNABLE TO ADD MARKETPLACE CUSTOM CODE TO PROJECT
UNABLE TO CREATE NEW PROJECT WITH CUSTOM MARKETPLACE APPS

Have over 20 apps and a freshly initiated Blank Project as well.
Triple checked with multiple accounts and multiple devices.
Have been running 150mbps of internet plan just to clear out any confusions regarding slow internet speed and loading time.
